MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -1369,21 +1342,24 @@ private void checkSalvageRights() {
 
     		// Set up the values
     		if(salvageBoxes.get(i).isSelected()) {
-    			salvageUnit += salvageables.get(i).getSellValue();
+    			salvageUnit = salvageUnit.plus(salvageables.get(i).getSellValue());
     		} else {
-    			salvageEmployer += salvageables.get(i).getSellValue();
+    			salvageEmployer = salvageEmployer.plus(salvageables.get(i).getSellValue());
     		}
     	}
-    	currentSalvagePct = (int)(100*((double)salvageUnit)/(salvageUnit+salvageEmployer));
+    	currentSalvagePct = salvageUnit.getAmount()
```

Comment:
same as above regarding newer Money overloads to avoid the back and forth with getAmount